src/HRayData.hs

```diff
@@ -0,0 +1,131 @@
+{-# LANGUAGE AllowAmbiguousTypes       #-}
+{-# LANGUAGE ExistentialQuantification #-}
+{-# LANGUAGE RecordWildCards           #-}
```

Comment:
This extension is very useful so you can safely put it into `default-extensions` section in `package.yaml`.